medusa/backup.py

```diff
@@ -57,10 +57,35 @@ def generate_md5_hash(src, block_size=BLOCK_SIZE_BYTES):
     return base64_md5
 
 
+def md5_multipart(src, part_size):
+    eof = False
+    hash_list = []
+    with open(str(src), 'rb') as f:
+        while eof is False:
+            (md5_hash, eof) = md5_part(f, part_size)
+            hash_list.append(md5_hash)
+
+    multipart_hash = hashlib.md5(b''.join(hash_list)).hexdigest()
+
+    return '%s-%d' % (multipart_hash, len(hash_list))
+
+
+def md5_part(f, count):
+    hash_md5 = hashlib.md5()
+    eof = False
+    for i in range(count * 16):
```

Comment:
After chatting with Alex, I understand this function is meant to have a cap on how much it reads, so it should not greedily go towards the end of the file.
If this is really so, I suggest we drop the `count` as an argument, and hard-code it together with the 16 and 65536, and perhaps hide behind some "constant" variables.

Reply:
done.